pyproject.toml

```diff
@@ -83,7 +83,7 @@ exclude=[
 # + E501 (line too long) because keeping it < 88 in cython
 # often makes code less readable.
 ignore = [
-    # check ignored by default in flake8. Meaning unclear.
+    # multiple spaces/tab after comma
```

Comment:
Have just put the rule here. Also I don't think it is ignored by default by [ruff](https://beta.ruff.rs/docs/configuration/#using-pyprojecttoml)